Framework/PythonInterface/plugins/algorithms/SelectNexusFilesByMetadata.py

```diff
@@ -52,6 +51,13 @@ def PyInit(self):
                              doc='Comma separated list of the fully resolved file names satisfying the given criteria.')
 
     def PyExec(self):
+
+        # run only if h5py is present
+        try:
+            import h5py
+        except ImportError:
+            raise RuntimeError('This algorithm requires h5py package')
```

Comment:
In other cases there's a link to the package on PyPI, which gives users a hint where to get the package. If h5py can be installed via pip, I think it would be a good idea to include such a link here (and maybe also in the documentation).